test/core/renderControllerTests.ts

```diff
@@ -0,0 +1,24 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("RenderController", () => {
+  it("Components whose render() is triggered by another Component's render() will be drawn", () => {
+    var link1 = new Plottable.Component();
+    var svg1 = TestMethods.generateSVG();
+    link1.anchor(svg1).computeLayout();
+    var link2 = new Plottable.Component();
+    var svg2 = TestMethods.generateSVG();
+    link2.anchor(svg2).computeLayout();
+
+    (<any> link1)._render = () => link2.render();
+    var link2Rendered = false;
+    (<any> link2)._render = () => link2Rendered = true;
+
+    link1.render();
+    assert.isTrue(link2Rendered, "dependent Component was eventually drawn");
+
+    svg1.remove();
+    svg2.remove();
+  });
```

Comment:
The test mostly looks good.  I think ideally you should be above to connect more directly to the `RenderController` so that we have a much more unitized test, but that said if you think this test is suitable enough, it's a good concise test.